### ISSUE TITLE
Getting the revision of a document

### ISSUE THREAD

**Issue by feo-elektronik-may** [jesper-raemaekers/python-polarion#173]
Is there a way to get the revision of a LiveDoc document?
The `Document` class does not offer a `getRevision()` function and I couldn't find anything else in the documentation.

**Comment by You69Man** [comment]
I'm also looking for a solution to get the current revision of a live document. Any idea by someone?

**Comment by feo-elektronik-may** [comment]
I did some investigating into the Polarion API and it looks like:

The [getModuleByLocation](https://testdrive.polarion.com/polarion/sdk/doc/javadoc/com/polarion/alm/ws/client/tracker/TrackerWebService.html#getModuleByLocation(java.lang.String,java.lang.String)) API does not provide a revision in the resulting [Module](https://testdrive.polarion.com/polarion/sdk/doc/javadoc/com/polarion/alm/ws/client/types/tracker/Module.html).

The [getRevisions](https://testdrive.polarion.com/polarion/sdk/doc/javadoc/com/polarion/alm/ws/client/tracker/TrackerWebService.html#getRevisions(java.lang.String)) API could be used to get all revisions based on the URI, which should be part of the "module" / document we already have.

To summarize, I think you'd need to implement the `getRevisions` API in the python-polarion client to be able to access a list of all revisions of any resource and you should be able to use this list of revisions to figure out the latest revision.
